### PR TITLE
etcdctl: fix migrate in outputing client.Node to json

### PR DESCRIPTION
**Pull request by hongchaodeng**
We should use print/println instead of printf.

### CODE REVIEW

**Comment by hongchaodeng** [comment]
cc @timothysc

**Comment by hongchaodeng** [comment]
@wojtek-t FYI

**Comment by hongchaodeng** [comment]
@xiang90 @gyuho 

This should be backport-ed to 3.0?

**Comment by gyuho** [comment]
Only difference is the `\n` at the end of output. Did it break anything in Kubernetes? Defer to @xiang90

**Comment by timothysc** [comment]
@hongchaodeng Did this fix the migrate failure?

**Comment by wojtek-t** [comment]
@timothysc - what migration failure are you talking about? If there is a bug, this should be backported to 3.0

**Comment by timothysc** [comment]
@wojtek-t yes we found a bug in yesterday on a large migrate we were testing.

**Comment by timothysc** [comment]
verified, this fixes the migration issue we came across.

File: etcdctl/ctlv3/command/migrate_command.go [hunk 1]
```diff
@@ -220,7 +220,7 @@ func writeKeys(w io.Writer, n *store.NodeExtern) uint64 {
 	if err != nil {
 		ExitWithError(ExitError, err)
 	}
-	fmt.Fprintf(w, string(b))
+	fmt.Fprintln(w, string(b))
```
Comment (xiang90) [inline etcdctl/ctlv3/command/migrate_command.go]: @hongchaodeng can you explain why do we need to change fprintf to fprintln in the commit message in detail?

Comment (hongchaodeng) [inline etcdctl/ctlv3/command/migrate_command.go]: Updated

Comment (xiang90) [inline etcdctl/ctlv3/command/migrate_command.go]: I read through https://golang.org/src/fmt/print.go. If seems the only difference is that println adds a `\n`. do you have an example?

Comment (hongchaodeng) [inline etcdctl/ctlv3/command/migrate_command.go]: Example for what?

Comment (xiang90) [inline etcdctl/ctlv3/command/migrate_command.go]: Also this should be https://golang.org/pkg/fmt/#Fprint not https://golang.org/pkg/fmt/#Fprintln

Comment (hongchaodeng) [inline etcdctl/ctlv3/command/migrate_command.go]: > how fprintf and fprintln behaviors differently in this case.

```
printf("%") => %!(MISSING)
print("%") => %
```

> Also this should be https://golang.org/pkg/fmt/#Fprint not https://golang.org/pkg/fmt/#Fprintln

Why print? Why not println?

Comment (xiang90) [inline etcdctl/ctlv3/command/migrate_command.go]: @hongchaodeng println adds a `\n` at the end of the line. why do we need that `\n`?

Comment (hongchaodeng) [inline etcdctl/ctlv3/command/migrate_command.go]: println is easier for user to debug on the other end.

Comment (xiang90) [inline etcdctl/ctlv3/command/migrate_command.go]:
```
printf("%") => %!(MISSING)
print("%") => %
```
 can you also add this into commit message?

**Comment by hongchaodeng** [comment]
@xiang90 Fixed

**Comment by xiang90** [comment]
lgtm

**Comment by wojtek-t** [comment]
@mml - FYI

@xiang90 @hongchaodeng - when can we expect backport of this to 3.0 branch?

**Pull request merged**